build-system/tasks/bundle-size/index.js

```diff
@@ -53,23 +57,36 @@ const bundleSizeAppBaseUrl = 'https://amp-bundle-size-bot.appspot.com/v0/';
  * @return {Map<string, number>} the bundle size in KB rounded to 2 decimal
  *   points.
  */
-function getBrotliBundleSizes() {
-  // Brotli compressed size fluctuates because of changes in the RTV number, so
-  // normalize this across pull requests by replacing that RTV with a constant.
+async function getBrotliBundleSizes() {
   const bundleSizes = {};
+  const replacementExpression = new RegExp(internalRuntimeVersion, 'g');
```

Comment:
micro-optimization: this can be a module-level const

Reply:
Will hoist.